<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpidrDummyGenDialog</class>
 <widget class="QDialog" name="SpidrDummyGenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SPIDR-TPX3 Dummy Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="dummygen.qrc">
    <normaloff>:/icon/spidr.ico</normaloff>:/icon/spidr.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBoxConnection">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>4</y>
     <width>354</width>
     <height>79</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection</string>
   </property>
   <widget class="QLabel" name="labelPeriod">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>34</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAddr0">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAddr2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>168</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="_pushButtonConnectOrDisconnect">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>29</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAddr1">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPeriod_3">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>34</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelIpAddr">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>14</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP address</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>14</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPeriod_2">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>34</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAddr3">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditPort">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_labelDisconnected">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>55</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>DISCONNECTED</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxDummyGen">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>90</y>
     <width>354</width>
     <height>159</height>
    </rect>
   </property>
   <property name="title">
    <string>Dummy Generator</string>
   </property>
   <widget class="QLineEdit" name="_lineEditFeBlockControl">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>120</y>
      <width>93</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_pushButtonStartOrStop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>45</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRegVal">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>119</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Register 0x210</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="_comboBoxHeader">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>84</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>16</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="_spinBoxDelay">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>21</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="_spinBoxFrames">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>52</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelHeader">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>85</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Header</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFrames">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>55</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Frames</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDelay">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>24</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Delay</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_labelFailed">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>* FAILED *</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="dummygen.qrc"/>
 </resources>
 <connections/>
</ui>
